lib/fs.js

```diff
@@ -228,7 +228,16 @@ function existsSync(path) {
     return false;
   }
   const ctx = { path };
-  binding.access(pathModule.toNamespacedPath(path), F_OK, undefined, ctx);
+  const nPath = pathModule.toNamespacedPath(path);
+  binding.access(nPath, F_OK, undefined, ctx);
+
+  // If it is an invalid symlink, existsSync should return false
+  // binding.access checking is not enough for win32 (return true in this case)
+  // So we need to call binding.stat for double checking
```

Comment:
Nit: perhaps this will be a bit clearer.
```
// In case of an invalid symlink `binding.access()` on win32
// will **not** return an error and is therefore not enough.
// Double check with `binding.stat()`.
```

Reply:
+1. Updated